.github/workflows/test-and-check.yml

```diff
@@ -102,10 +102,21 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-13]
         filter: ["./packages/*", "./fixtures/*"]
+        node_version: [18]
```

Comment:
I think CI is pulling something like v18.20.2 which is too old - might need to make these more specific.

Reply:
I've loosened this up a bit to only require v18